jobs/config.json

```diff
@@ -8684,47 +8684,6 @@
       "sig-node"
     ]
   },
-  "ci-kubernetes-pull-gce-federation-deploy": {
```

Comment:
This job is required currently and is the daily cluster recycler for federation presubmit job.